src/components/Modal/Modal.vue

```diff
@@ -109,13 +109,13 @@
 			:class="{ 'modal-mask--dark': dark }"
 			:style="cssVariables"
 			role="dialog"
-			aria-labelledby="modal-title"
-			aria-describedby="modal-description">
+			:aria-labelledby="'modal-title-' + randId"
```

Comment:
Just a personal preference:
```suggestion
			:aria-labelledby="`modal-title-${randId}`"
```
and maybe make it a computed prop?